include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -485,10 +489,10 @@ struct OrtApi {
 
   /** \brief Create an OrtEnv
   *
-  * \param[in] logging_function
-  * \param[in] logger_param
-  * \param[in] logging_level
-  * \param[in] logid
+  * \param[in] logging_function A user-provided logging function.
```

Comment:
All parameters are user provided, maybe 'a pointer to a function called on every logging event'?